dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/model/ResourceLink.java

```diff
@@ -44,21 +51,26 @@
 
     private final String mimeType;
 
-    private final FileAsset fileAsset;
+    private final Contentlet fileAsset;
+
+    private final String fieldVar;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'fieldVar' could be replaced by a local variable.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5372524)